src/components/WorkspaceDialog.vue

```diff
@@ -8,7 +8,7 @@
     origin="center center"
     transition="scale-transition"
   >
-    <template v-slot:activator>
+    <template v-slot:activator="{ }">
```

Comment:
The reason I believe this is happening, is because currently the whole dialog is wrapped in a `v-menu`, which is providing the "Getting Started" popover. This should probably go into `Sidebar.vue` or somewhere more appropriate, but regardless probably shouldn't be wrapping whole components like this. I say we fix this in a follow-up issue.

Reply:
Can you file an issue and coordinate with @jtomeck? I don't seem to understand what's going wrong here as well as you do.